backend/fault.py

```diff
@@ -12,53 +11,39 @@ def __init__(self, message):
 
 
 class Fault:
-    def __init__(self, tlc, severity, rank, level, suffix, okValue, faultValue,
-                 rack, description, name):
+    def __init__(self, tlc, severity, suffix, okValue, faultValue, description,
+                 name, prefix):
         self.tlc = tlc
         self.severity = int(severity)
-        self.rank = rank
-        self.level = level
-        self.rack = rack
-        self.suffix = suffix
         self.description = description
         self.name = name
         self.okValue = float(okValue) if okValue else None
         self.faultValue = float(faultValue) if faultValue else None
 
+        self.pv: PV = PV(prefix + ":" + suffix, connection_timeout=PV_TIMEOUT)
+
     def __str__(self):
         return ', '.join("%s: %s" % item for item in vars(self).items())
```

Comment:
I forgot this def __str__(self) line was in our code. Do we actually use it anywhere?

Reply:
Nope we don't use it, I can delete it!